### PR TITLE
Ldapadmin export users

### PR DESCRIPTION
**Pull request by vdorut**
Ldapadmin users export to Vcard and CSV with outlook header.

### CODE REVIEW

File: ldapadmin/src/main/webapp/privateui/js/controllers.js [hunk 1]
```diff
@@ -198,23 +198,50 @@ angular.module('ldapadmin.controllers', [])
...
+          window.location = '#/users';
+          flash.success = 'Group correctly removed';
+        }, function errorCallback() {
+          flash.error = 'Error removing the group';
+        });
       }
     };
+
+    $scope.exportAsCsv = function() {
+      var uids = _.pluck($scope.selectedUsers(), 'uid');
```
Comment (fvanderbiest) [inline ldapadmin/src/main/webapp/privateui/js/controllers.js]: nice construct :-)

File: ldapadmin/src/main/webapp/privateui/js/controllers.js [hunk 2]
```diff
@@ -198,23 +198,50 @@ angular.module('ldapadmin.controllers', [])
...
+      var url = GEOR_config.publicContextPath + "private/users.csv";
+  
+      var form = $('<form> </form>').attr("action", url).attr("method", 'POST').attr("id", '#exportAsCsvForm');
+      var input = $('<input >').attr("name", 'users');
+  
+      $(document).ready(function() {
```
Comment (fvanderbiest) [inline ldapadmin/src/main/webapp/privateui/js/controllers.js]: Is this required ? (I mean ```$(document).ready(... ```)

**Comment by fvanderbiest** [comment]
Very nicely done, thanks a lot Vincent + Pierre !

**Comment by fvanderbiest** [comment]
Deployed to sdi.georchestra.org, where the button action points to https://sdi.georchestra.org/ldapadminprivate/users.csv ... missing ```/``` somewhere ?

(In addition, we need to migrate the ldap groups on this platform)

**Comment by pmauduit** [comment]
> missing / somewhere ?

Done

**Pull request merged**